.github/workflows/validate.yml

```diff
@@ -148,6 +148,17 @@ jobs:
         make cfmt
         git diff --exit-code
 
+  gosec:
+    runs-on: ubuntu-20.04
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Run Gosec Security Scanner
+      uses: securego/gosec@master
+      with:
+        args: -confidence medium -severity high -exclude-dir libcontainer/nsenter ./...
```

Comment:
Since we already run golangci-lint, which supports gosec, perhaps it's better to implement these changes in `.golangci.yml` instead of adding a separate CI job. I'm sure it's possible to add a linter-specific configuration in there.